### PR TITLE
Paas 1977 jcustomer duplicate events

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1977

Short description:

### CODE REVIEW

File: packages/jcustomer/migrations/migrate-to-v11.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ onInstall:
   - setJelEnvRoleInNodeGroupData # PAAS-1959
   - configureShardsAndReplicas   # PAAS-1992
   - migrateJcustomer2EC          # PAAS-1970
+  - fixjCustomerEvents           # PAAS-1977
```
Comment (kevinvandel) [inline packages/jcustomer/migrations/migrate-to-v11.yaml:29]: Question: shouldn't we perform this _before_ the "eventsUpdate" action ?

File: packages/jcustomer/migrations/migrate-to-v11.yaml [hunk 2]
```diff
@@ -356,6 +357,57 @@ actions:
           type: error
           message: "envmode value is wrong on ${env.envName}"
 
+  fixjCustomerEvents:
+    - setGlobals:
+        jel_dashboard_url: ${platformUrl}
```
Comment (Ronan-Gorain) [inline packages/jcustomer/migrations/migrate-to-v11.yaml:362]: This variable seems to be unused

Comment (adoux) [reply packages/jcustomer/migrations/migrate-to-v11.yaml]: Good call! Deleted

File: packages/jcustomer/migrations/migrate-to-v11.yaml (hunk 1 above)
Comment (adoux) [reply packages/jcustomer/migrations/migrate-to-v11.yaml]: Nope, events are linked to an installedApp, so when we update the events with the new JPS id, it duplicates events to and associate the new ones with the new installedApp.
When we delete the previous one, it only deletes events that are associated to the previous installedApp

**Review by Ronan-Gorain — APPROVED** [review]

**Pull request merged**